SS_benchfore.tpl

```diff
@@ -1664,14 +1668,14 @@ FUNCTION void Get_Benchmarks(const int show_MSY)
 
     Btgt_Fmult2 = Fmult;
     if (rundetail > 0 && mceval_counter == 0 && show_MSY == 1)
-      cout << " got_F_Blimit " << Btgt_Fmult2 << " " << Btgt2 / Blim_report << endl;
+      echoinput << "Calculated F_Blimit " << Btgt_Fmult2 << " " << Btgt2 / Blim_report << endl;
```

Comment:
Needs curly braces.

Reply:
I was mis-reading these. sorry. It is a single statement and brackets/braces are not required. However, I do like to use them.